DD4.16.3 release notes

aevesdocker opened the pull request.

aevesdocker commented on desktop/release-notes.md:
```diff
@@ -24,13 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a> 
+        <br>
+        <br>
+        <b>Checksum:</b> 
```
To do

aevesdocker commented on desktop/release-notes.md:
```diff
@@ -24,13 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+
+#### For Windows
+
+
```
For you @mat007 - what needs to be added here?

mat007 commented on desktop/release-notes.md:
```diff
@@ -24,13 +24,76 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+
+#### For Windows
+
+
```
Something like:
```suggestion
- Fixed Docker Desktop failing to start the WSL 2 backend when file system compression or encryption is enabled on `%LOCALAPPDATA%`. Fixes [docker/for-win#13184](https://github.com/docker/for-win/issues/13184).
- Fixed Docker Desktop failing to report a missing or outdated WSL installation when starting. Fixes [docker/for-win#13184](https://github.com/docker/for-win/issues/13184).
```

aevesdocker replied:
Perfect 👍

mat007 reviewed (commented):
What about `_includes/desktop-install.html`? Should we comment out all but Windows in there?

aevesdocker commented:
https://deploy-preview-16599--docsdocker.netlify.app/desktop/release-notes/ I removed the includes file and just added the Windows accordion. Looks the same in the preview

mat007 commented on desktop/release-notes.md:
```diff
@@ -24,13 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a> 
+        <br>
+        <br>
+        <b>Checksum:</b> 
```
```suggestion
        <b>Checksum: b9758c302b4bddd661a4b2307d9fa3f85658b5e69b3f11a0586f3dc32c524386</b> 
```

mat007 commented:
> I removed the includes file and just added the Windows accordion. Looks the same in the preview

Oh right! Will we keep these inline from now on?

aevesdocker commented:
| Will we keep these inline from now on?

We could? But it makes the release-notes.md look extra complex when editing it. Having it in the includes file keeps everything neat, I think.

mat007 commented on desktop/release-notes.md:
```diff
@@ -24,13 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
+## 4.16.3
+
+{% include release-date.html date="2023-01-26" %}
```
Delta updates are still being created, we should probably postpone releasing until tomorrow.
```suggestion
{% include release-date.html date="2023-01-27" %}
```

mat007 commented on desktop/release-notes.md:
```diff
@@ -24,13 +24,77 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a> 
+        <br>
+        <br>
+        <b>Checksum: b9758c302b4bddd661a4b2307d9fa3f85658b5e69b3f11a0586f3dc32c524386</b> 
```
There is actually a new build now:
```suggestion
        <b>Checksum: 99f6600e7a05c7e56490600497b64a5039eb15f3413c1cf7b3a89bdfc4d947b8</b> 
```

aevesdocker commented:
What time are you releasing @mat007
